src/SDKs/DevSpaces/Management.DevSpaces/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd devspaces/resource-manager %*
```

Comment:
Please replace this with a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Monitor/Management.Monitor/generate.ps1) and regenerate the code.
Please commit any .txt files that get generated.